### ISSUE TITLE
knctl uninstall don't delete mutatingwebhookconfiguration

### ISSUE THREAD

**Issue by den-vasyliev** [cppforlife/knctl#51]
First, thank you, Dmitriy for amazing toolset!

Please find issue description:

#kn install
...
Waiting for Knative to start...

Succeeded

#k get mutatingwebhookconfiguration 
NAME                          AGE
istio-sidecar-injector        59s
webhook.build.knative.dev     20s
webhook.serving.knative.dev   13s

#kn uninstall
Uninstalling Knative Build
Waiting for namespace 'knative-build' to be deleted...
Uninstalling Knative Serving
Waiting for namespace 'knative-serving' to be deleted...
Uninstalling Knative Monitoring
Waiting for namespace 'knative-monitoring' to be deleted...
Uninstalling Istio
Waiting for namespace 'istio-system' to be deleted...

Succeeded

#k get mutatingwebhookconfiguration 
NAME                     AGE
istio-sidecar-injector   7m

#k run nginx --image nginx
deployment.apps "nginx" created

#k get deployment nginx
NAME      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
nginx     1         0         0            0           8s

#k describe rs nginx-65899c769f 
...
Events:
  Type     Reason        Age                 From                   Message
  ----     ------        ----                ----                   -------
  Warning  FailedCreate  16s (x14 over 58s)  replicaset-controller  Error creating: Internal error occurred: failed calling admission webhook "sidecar-injector.istio.io": Post https://istio-sidecar-injector.istio-system.svc:443/inject?timeout=30s: service "istio-sidecar-injector" not found

 #k delete mutatingwebhookconfiguration --all
mutatingwebhookconfiguration.admissionregistration.k8s.io "istio-sidecar-injector" deleted

#k run nginx --image nginx
deployment.apps "nginx" created

#k get deployment nginx
NAME      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
nginx     1         1         1            1           10s

**Comment by cppforlife** [comment]
@den-vasyliev yup, i noticed that a few cluster level resources are left around as well. i've been writing a separate generic tool/library to make management of resources easier. will report back once that's open sourced.

**Comment by cppforlife** [comment]
> will report back once that's open sourced.

tool is now open sourced: https://get-kapp.io/. i'll probably be removing install/uninstall commands from knctl at some point.
